src/commands/runtime/trigger/create.js

```diff
@@ -45,6 +45,9 @@ class TriggerCreate extends RuntimeBaseCommand {
       if (Object.keys(annotationParams).length) {
         triggerParams.annotations = annotationParams
       }
+      if (flags.feed) {
```

Comment:
what about:
1. --feed-param
2. --trigger-param
(see `wsk trigger create --help`)

Reply:
Suggest pushing off the two flags above into a new PR, if needed. Seems to me these are fairly new in 2020, discussion here: https://github.com/apache/openwhisk-cli/issues/293#issuecomment-569975564

Reply:
I'm not even sure if they are supported in the wskdeploy manifest format even.

Reply:
The approach SGTM.